gm4_display_frames/data/gm4_display_frames/predicates/invisibility_potion_marker.json

```diff
@@ -0,0 +1,26 @@
+{
+  "condition": "minecraft:alternative",
+  "terms": [
+    {
+      "condition": "minecraft:entity_properties",
+      "entity": "this",
+      "predicate": {
+        "nbt": "{data:{gm4_potion:{Item:{tag:{Potion:'minecraft:invisibility'}}}}}"
```

Comment:
This predicate doesn't help to optimize these checks. The NBT checks still need to happen and the entity data will be serialized 3 times. It will be faster to check storage or entity data with commands.